packages/frameworks/src/frameworks.ts

```diff
@@ -2006,6 +2006,51 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Sanity',
+    slug: 'sanity',
+    demo: 'https://www.sanity.io/get-started',
+    logo: 'https://cdn.sanity.io/images/3do82whm/next/cd02ce807fbd324cbda82a77cd726c10ff6074d1-128x128.svg',
```

Comment:
Can you copy this file into the `frameworks/logos` directory and have it reference the GitHub URL like the other ones do?

Reply:
Done!